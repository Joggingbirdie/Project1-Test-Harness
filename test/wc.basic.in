This is a basic test.
It has multiple lines.
The word count should be accurate.
