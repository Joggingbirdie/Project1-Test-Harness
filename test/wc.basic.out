3    17    99
